allennlp_models/generation/models/simple_seq2seq.py

```diff
@@ -66,21 +69,26 @@ class SimpleSeq2Seq(Model):
     """
 
     def __init__(
-        self,
-        vocab: Vocabulary,
-        source_embedder: TextFieldEmbedder,
-        encoder: Seq2SeqEncoder,
-        max_decoding_steps: int,
-        attention: Attention = None,
-        beam_size: int = None,
-        target_namespace: str = "tokens",
-        target_embedding_dim: int = None,
-        scheduled_sampling_ratio: float = 0.0,
-        use_bleu: bool = True,
-        bleu_ngram_weights: Iterable[float] = (0.25, 0.25, 0.25, 0.25),
+            self,
+            vocab: Vocabulary,
+            source_embedding_dim: int,
+            source_embedder: TextFieldEmbedder,
+            encoder: Seq2SeqEncoder,
+            max_decoding_steps: int,
+            attention: Attention = None,
+            beam_size: int = None,
+            decoder_layers: int = 2,
```

Comment:
Should this be called `target_decoder_layers`, and default to 1?

Reply:
Finish